src/pages/index.js

```diff
@@ -36,82 +82,213 @@ const editCardPopup = new PopupWithForm(
 );
 editCardPopup.setEventListeners();
 
-const userInfo = new UserInfo({
-  nameSelector: ".profile__title",
-  jobSelector: ".profile__description",
-});
-
-const cardSection = new Section(
+const editAvatarPopup = new PopupWithForm(
   {
-    items: initialCards,
-    renderer: (cardData) => {
-      const cardElement = createCard(cardData); // Create card using createCard
-      cardSection.addItem(cardElement); // Add the card to the DOM
-    },
+    popupSelector: "#avatar-edit-modal",
   },
-  ".cards__list" // Container selector
+  handleEditAvatarSubmit
 );
+editAvatarPopup.setEventListeners();
 
 const popupWithImage = new PopupWithImage({
   popupSelector: "#previewModal",
 });
 popupWithImage.setEventListeners();
 
+const confirmPopup = new ConfirmPopup({
+  popupSelector: "#delete-modal",
+});
+confirmPopup.setEventListeners();
+
+// Instantiate form validators
 const editFormValidator = new FormValidator(settings, profileEditForm);
 const addFormValidator = new FormValidator(settings, addCardForm);
+editFormValidator.enableValidation();
+addFormValidator.enableValidation();
+
+// Helper Functions
+function createCard({ name, link, _id, isLiked }) {
+  // Instantiate the card
+  const card = new Card(
+    { name, link, _id, isLiked },
+    "#card-template",
+    handleImageClick,
+    handleLikeClick,
+    handleUnlikeClick,
+    handleDeleteCard
+  );
+
+  const cardElement = card.getView();
+  const cardContainer = document.querySelector(".modal__container");
+
+  cardContainer.appendChild(cardElement);
 
-// Render initial cards
-cardSection.renderItems();
+  const deleteButton = cardElement.querySelector(".card__delete-button");
 
-// Functions
+  if (deleteButton) {
+    deleteButton.addEventListener("click", () => {
+      confirmPopup.open();
+    });
+  } else {
+    console.error("Delete button not found within card template.");
```

Comment:
Please, delete it


Now only class `Card` knows every **card selector**, so you need to encapsulate the likes/deleting logics inside its methods and call them in `index.js`. It’s a violation of **Single responsibility** principle


You pass `handleDeleteCard` into the constructor. It's the delete handler where you should open the confirmation modal